### PR TITLE
feat(amplify-category-api): change default graphql query limit to 100

### PR DESCRIPTION
**Pull request by attilah**
*Description of changes:*

For generated GraphQL queries for types, ```@searchable``` and ```@connection``` directive the default item count was 10, this default limit has been raised to 100, without specifying the ```limit``` parameter DynamoDB and ElasticSearch is queried for 100 items.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/graphql-transformer-common/src/ResourceConstants.ts [hunk 1]
```diff
@@ -1,6 +1,10 @@
 export class ResourceConstants {
   public static NONE = 'NONE';
 
+  public static DEFAULT_PAGE_LIMIT = 100;
+  public static DEFAULT_DATASYNC_PAGE_LIMIT = 100;
```
Comment (yuth) [inline packages/graphql-transformer-common/src/ResourceConstants.ts:5]:
```suggestion
  public static DEFAULT_SYNC_QUERY_PAGE_LIMIT = 100;
```

**Comment by smithad15** [comment]
I just ran into this change in functionality within my application while running `amplify mock` after an upgrade to version `4.21.1` of `@aws-amplify/cli`. While I agree that the initial limit of 10 seemed overly restrictive, I can't seem to find where this change is documented in a changelog? Could you please point me in the right direction as I am having a very difficult time getting a clear holistic picture of what has changed due to the nature of the mono-repo. While it was of minor impact for me, this could have serious breaking changes to expected application functionality for others.

**Comment by attilah** [comment]
@smithad15 until we'll have a unified change log you can see the changes individually for each package:
- https://github.com/aws-amplify/amplify-cli/blob/master/packages/graphql-dynamodb-transformer/CHANGELOG.md#6170-2020-05-08
- https://github.com/aws-amplify/amplify-cli/blob/master/packages/graphql-key-transformer/CHANGELOG.md#2160-2020-05-08
- https://github.com/aws-amplify/amplify-cli/blob/master/packages/graphql-elasticsearch-transformer/CHANGELOG.md#460-2020-05-08
- https://github.com/aws-amplify/amplify-cli/blob/master/packages/graphql-connection-transformer/CHANGELOG.md#4160-2020-05-08

Changes also reflected in the docs: https://docs.amplify.aws/cli/graphql-transformer/directives

**Comment by smithad15** [comment]
Thank you! I hope you are eventually able to put together a unified change log for the cli package so that everyone can track the great work you are doing a little simpler. I'm loving how the Amplify ecosystem is evolving

**Pull request merged**